db/base/templatetags/tags.py

```diff
@@ -17,6 +17,6 @@ def frq(value):
         to_format = float(value)
     except (TypeError, ValueError):
         return ''
-    formatted = format(float(to_format) / 1000000, '.3f')
+    formatted = format(float(to_format) / 1000000, '.4f')
```

Comment:
Nice idea. I took a try (see bellow). I think I prefer the small digit. I'll leave the tooltip, in case we need to see the full freq.